app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
+        try:
+            cafe.visit_cafe(friends[i])
+        except (NotVaccinatedError, OutdatedVaccineError):
+            return 'All friends should be vaccinated'
```

Comment:
Ensure that the exception messages in visit_cafe method are informative enough for debugging purposes.
It would be better to add names that aren't vaccinated